src/Uno.UWP/UI/ViewManagement/ApplicationView.cs

```diff
@@ -46,7 +48,21 @@ public bool SetDesiredBoundsMode(global::Windows.UI.ViewManagement.ApplicationVi
 			return true;
 		}
 
-		public Foundation.Rect VisibleBounds { get; internal set; }
+		public Foundation.Rect VisibleBounds { get; private set; }
+
+		/// <summary>
+		/// All other platforms: equivalent to <see cref="VisibleBounds"/>.
+		///
+		/// Android: returns the visible bounds taking the status bar into account. The status bar is not removed from <see cref="VisibleBounds"/>
+		/// on Android when it's opaque, on the grounds that the root managed view is already arranged below the status bar in y-direction by
+		/// default (unlike iOS), but in some cases the correct total height is needed, hence this property.
+		/// </summary>
+		internal Rect TrueVisibleBounds =>
```

Comment:
Should this be named `NativeVisibleBounds` ?

Reply:
I debated the name with myself and didn't find a good one... it's not really 'native' though. 

I suspect that the public `VisibleBounds` property should really be returning this value, but I wasn't sure enough to make the breaking change, and anyway it was out of scope for this PR. (It would require adjusting VisibleBoundsPadding obviously and potentially the Duo dual-screen control as well.)

Reply:
It's internal, so not a big deal :)